phpstan.neon

```diff
@@ -1,9 +1,10 @@
 includes:
+    - dev-tools/vendor/phpstan/phpstan/conf/bleedingEdge.neon
```

Comment:
Oh no 😢 why not in alphabetical order?

Reply:
I think the order matters?

Reply:
I does not change anything for us anyway. The order I chose was: PHPStan files first, extension files then. I changed for alphabetical order.